Makefile

```diff
@@ -83,4 +85,12 @@ get-ip:
 
 .PHONY: seed
 seed:
-	$(DOCKER_COMPOSE) exec -it transcendence-app npx knex seed:run
+	$(DOCKER_COMPOSE) exec transcendence-app npx knex seed:run
```

Comment:
I removed the `-it` flags because it doesn't work on campus `docker-compose` v1.